doc/release/1.6.0-notes.rst

```diff
@@ -15,66 +15,188 @@ optimizations. Before upgrading, we recommend that users check that
+
+A new boundary mode, ``grid-wrap`` was added which wraps images periodically,
+using a period equal to the shape of the input image grid. This is in contrast 
+to the existing ``wrap`` mode which uses a period that is one sample smaller 
+than the original signal extent along each dimension.
+
+A long-standing bug in the ``reflect`` boundary condition has been fixed and 
+the mode ``grid-mirror`` was introduced as a synonym for ``reflect``.
+
+A new boundary mode, ``grid-constant`` is now available. This is similar to 
+the existing ndimage ``constant`` mode, but interpolation will still performed 
+at coordinate values outside of the original image extent. This 
+``grid-constant`` mode is consistent with OpenCV's ``BORDER_CONSTANT`` mode 
+and scikit-image's ``constant`` mode.
+
+Spline pre-filtering (used internally by ``ndimage`` interpolation functions 
+when ``order >= 2``), now supports all boundary modes rather than always 
+defaulting to mirror boundary conditions. The standalone functions 
+``spline_filter`` and ``spline_filter1d`` have analytical boundary conditions 
+that match modes ``mirror``, ``grid-wrap`` and ``reflect``.
+
+`scipy.ndimage.interpolation` functions now accept complex-valued inputs. In 
+this case, the interpolation is applied independently to the real and 
+imaginary components.
+
+The ``ndimage`` tutorials 
+(https://docs.scipy.org/doc/scipy/reference/tutorial/ndimage.html) have been 
+updated with new figures to better clarify the exact behavior of all of the 
+interpolation boundary modes.
+
+`scipy.ndimage.zoom` now has a ``grid_mode`` option that changes the coordinate 
+of the center of the first pixel along an axis from 0 to 0.5. This allows 
+resizing in a manner that is consistent with the behavior of scikit-image's 
+``resize`` and ``rescale`` functions (and OpenCV's ``cv2.resize``).
 
 `scipy.optimize` improvements
 -----------------------------
+`scipy.optimize.linprog` has fast, new methods for large, sparse problems from 
+the ``HiGHS`` C++ library. ``method='highs-ds'`` uses a high performance dual 
+revised simplex implementation (HSOL), ``method='highs-ipm'`` uses an 
+interior-point method with crossover, and ``method='highs'`` chooses between 
+the two automatically. These methods are typically much faster and often exceed 
+the accuracy of other ``linprog`` methods, so we recommend explicitly 
+specifying one of these three method values when using ``linprog``.
 
+`scipy.optimize.quadratic_assignment` has been added for approximate solution 
+of the quadratic assignment problem.
 
 `scipy.signal` improvements
 ---------------------------
+`scipy.signal.gammatone` has been added to design FIR or IIR filters that 
+model the human auditory system.
+
+`scipy.signal.iircomb` has been added to design IIR peaking/notching comb 
+filters that can boost/attenuate a frequency from a signal.
 
 
 `scipy.sparse` improvements
 ---------------------------
```

Comment:
Ah this PR just crossed wires with the `csgraph` entry added to the wiki - can you sync that?

Reply:
Yes, note to self--add this:

> A solver for the minimum weight full matching problem for bipartite graphs, also known as the linear assignment problem, has been added in scipy.sparse.csgraph.min_weight_full_bipartite_matching. In particular, this provides functionality analogous to that of scipy.optimize.linear_sum_assignment, but with improved performance for sparse inputs, and the ability to handle inputs whose dense representations would not fit in memory.